frontend/src/pages/user/UserPageView.jsx

```diff
@@ -0,0 +1,44 @@
+import { List, ListItem } from '@mui/material'
+import Button from '@mui/material/Button'
+import classes from './userpage.module.scss'
+import Footer from '../../components/layout/footer/FooterController'
+import HeaderCustom from '../../components/layout/headercustom/HeaderCustomController'
+import ProfileUserDetails from '../../components/user/profiledetails/ProfileUserDetailsController'
+import Post from '../../components/posts/post/PostController'
+import Divider from '@mui/material/Divider'
+
+// eslint-disable-next-line no-unused-vars
+const UserPageView = ({ userData, userFeed, loggedIn }) => (
+    <div className={classes.container}>
+        <HeaderCustom title={userData.username} />
+        <img
+            className={classes.banner}
+            alt="Profile Banner"
+            src={userData.profileBanner}
+        />
+        <div className={classes.pageContent}>
+            <Button
+                variant="contained"
+                disableElevation
+                className={classes.followButton}
+            >
```

Comment:
You should pass an empty onClick handler from the controller so we can easily add logic later